dbms/src/Storages/KVStore/ProxyStateMachine.h

```diff
@@ -367,33 +367,28 @@ struct ProxyStateMachine
         }
     }
 
-    // Set KVStore to running, so that it could handle read index requests.
-    void runKVStore(TMTContext & tmt_context) const { tmt_context.setStatusRunning(); }
-
-    /// Stop all services in TMTContext and ReadIndexWorkers.
-    /// Then, inform proxy to stop by setting `tiflash_instance_wrap.status`.
-    void stopProxy(TMTContext & tmt_context)
+    /// Stopping services in TMTContext and ReadIndexWorkers.
+    void stoppingProxy(TMTContext & tmt_context)
```

Comment:
How about naming it into something like `stopProxyReadingService`, so others can know we can't send any read request(namely read-index) to proxy?